dali/operators/image/remap/displacement_filter_impl_gpu.cuh

```diff
@@ -208,7 +219,10 @@ class DisplacementFilter<GPUBackend, Displacement,
   explicit DisplacementFilter(const OpSpec &spec) :
       Operator(spec),
       displace_(spec),
-      interp_type_(spec.GetArgument<DALIInterpType>("interp_type")) {
+      interp_type_(spec.GetArgument<DALIInterpType>("interp_type")),
+      flat_block_setup_(32),
+      channel_block_setup_(32) {
```

Comment:
Why not just put it in the member definition?

Reply:
No particular reason, but due to the intricacies of C++, I tried it and could not use `flat_block_setup_ = {32};` only, `FlatBlockSetup flat_block_setup_ = FlatBlockSetup(32);`. Not sure which one is nicer.

Reply:
Hmm, I can `flat_block_stup_{32}`.